packages/gestalt/src/icons/workflow-status-warning.svg

```diff
@@ -0,0 +1 @@
+<svg width="24" height="24" fill="none" xmlns="http://www.w3.org/2000/svg"><path d="M10.36 4.67c.77-.8 2.19-.9 3.06-.2l.22.2 5.79 5.98c.76.8.76 1.9 0 2.69l-5.79 5.99c-.77.8-2.19.9-3.06.2l-.22-.2-5.79-5.99c-.76-.79-.76-1.89 0-2.69zM12 15c-.6 0-1 .4-1 1s.4 1 1 1 1-.4 1-1-.4-1-1-1zm0-7c-.6 0-1 .4-1 1v4c0 .6.4 1 1 1s1-.4 1-1V9c0-.6-.4-1-1-1z" fill="#8e8e8e"></path></svg>
```

Comment:
Please be sure to use an svg compression tool when adding icons like https://jakearchibald.github.io/svgomg/ Since this SVG is simple the only difference is to change `<path ...></path>` to `<path/>` but it's a good habit to always do this step first.

Reply:
Thanks for showing me this! Just updated it.